Sources/Screens/Home/TabbedViewController.swift

```diff
@@ -69,6 +65,16 @@ final class TabbedViewController: BaseViewController {
             tableView.tableHeaderView = headerView
         }
     }
+    
+    private func configureTableView() {
+        tableView.register(TabbedTableViewCell.self, forCellReuseIdentifier: TabbedTableViewCell.identifier)
+        tableView.register(TabbedViewSectionFooter.self, forHeaderFooterViewReuseIdentifier: TabbedViewSectionFooter.identifier)
+        tableView.register(TabbedViewSectionHeader.self, forHeaderFooterViewReuseIdentifier: TabbedViewSectionHeader.identifier)
+        tableView.tableHeaderView = headerView
+        tableView.sectionFooterHeight = UITableView.automaticDimension
```

Comment:
Is this needed if we have `tableView(_:heightForFooterInSection:)`? https://developer.apple.com/documentation/uikit/uitableview/1614846-sectionfooterheight

Reply:
We have heightForHeaderInSection, not heightForFooter.  Regardless, it seemed necessary for making sure the footer correctly responded to accessibility size changes.  I didn't do the same for the header because as it is not wrapping text, it worked better to use the intrinsic content size.  I confess to not having a strong understanding of the nuances of the sizing behaviors.